docs/schema/hashtablesource-element-workflowinfo.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: HashtableSource Element (WorkflowInfo)
+description: HashtableSource Element (WorkflowInfo) specifies that the evaluation value is retrieved from a hash table.
 manager: laurawi
 ms.date: 3/9/2015
```

Comment:
When you submit a change to a doc, please update the `ms.date` front matter property to the format `MM/DD/YYYY` to reflect the date of the change.